scripts/xtensa-build-zephyr.py

```diff
@@ -302,22 +278,60 @@ def check_west_installation():
 			"https://docs.zephyrproject.org/latest/getting_started/index.html")
 	print(f"Found west: {west_path}")
 
+def west_reinitialize(west_root_dir: pathlib.Path, west_manifest_path: pathlib.Path):
+	"""[summary] Performs west reinitialization to SOF manifest file asking user for permission.
+	Prints error message if script is running in non-interactive mode.
+
+	:param west_root_dir: directory where is initialized.
+	:type west_root_dir: pathlib.Path
+	:param west_manifest_path: manifest file to which west is initialized.
+	:type west_manifest_path: pathlib.Path
+	:raises RuntimeError: Raised when west is initialized to wrong manifest file
+	(not SOFs manifest) and script is running in non-interactive mode.
+	"""
+	global west_top
+	message = "West is initialized to manifest other than SOFs!\n"
+	message +=  f"Initialized to manifest: {west_manifest_path}." + "\n"
+	west_workspace_directory = pathlib.Path(west_root_dir.resolve(), ".west")
```

Comment:
This is not the "workspace directory". The "workspace" is the entire set of git repos, this is described in the intro to `west`: https://docs.zephyrproject.org/latest/develop/west/basics.html

Call it `dot_west_directory` if you really need a name for it.

Reply:
Ok will rename